compose/production/django/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6-alpine
+FROM public.ecr.aws/alexaprizesharedresources/python:3.7-slim-buster
```

Comment:
Can you use a non slim buster for now, and once you get a successful deploy, switch to slim buster?

Reply:
also preferrably use a verified account like bitnami https://gallery.ecr.aws/bitnami/python